docs/concepts/cluster-administration/proxies.md

```diff
@@ -20,6 +20,7 @@ There are several different proxies you may encounter when using Kubernetes:
     - proxy to apiserver uses HTTPS
     - locates apiserver
     - adds authentication headers
+    - depends on the conntrack package on Linux to function properly
```

Comment:
can we actually remove this? It's an important point---but notice how all the descriptions of the proxies on this page focus on what they *do*, as opposed to what they *need* to run. 

I think it's sufficient to have it on the other pages you've edited.